database/migrations/2014_10_12_000000_create_users_table.php

```diff
@@ -1,5 +1,7 @@
 <?php
 
+declare(strict_types=1);
```

Comment:
If you're gonna do this (which I like) - then probably remove the comments and add void return types too